scripts/unit.sh

```diff
@@ -3,7 +3,7 @@
 source ${OKTA_HOME}/${REPO}/scripts/setup.sh
 
 export TEST_SUITE_TYPE="jsunit"
-export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/jasmine"
+export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/"
```

Comment:
- looks like you didn't enable jest-junit yet. would be a result file generated?

Reply:
looks like it doesn't match jest junit config `./build2/reports/unit/junit-result.xml`.
or maybe simplify the config to be `./build2/reports/jest-junit-result.xml`.

Reply:
We're outputting two separate directories within `build2/reports`:

- `lint`: Results from `npm run lint:report`
- `unit`: Results from `npm run test:report`

I'll update this to be clear which directory we're outputting to:

```
export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/unit"
```